<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>505</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>15000</width>
    <height>15000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RunAsGPU</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Application</string>
    </property>
   </widget>
   <widget class="QListView" name="appList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>641</width>
      <height>281</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApplicationAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Application</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUnitSelector">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Select GPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnitSelected">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected GPU: &lt;b&gt;&lt;i&gt;{{UNIT}}&lt;/i&gt;&lt;/b&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>315</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filterBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>591</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Application Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemove">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove App</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
